docs/NEL.md

```diff
@@ -139,6 +139,14 @@ NEL[A].filter(fn: (element: A) => Boolean): List[A]
 
 Returns a new list, keeping only elements for which the predicate returns true.
 
+### filterNot
+
+```scala
+NEL[A].filterNot(fn: (element: A) => Boolean): List[A]
```

Comment:
`A => Boolean`

Reply:
This happens in many places. I can change it but I'd rather it in a separate pull request for clarity.

Reply:
I mean `.filterNot(fn: A => Boolean)`, but if there is more `(x: A) => B` then :+1: for separate PR.